scala_proto/BUILD

```diff
@@ -83,6 +83,7 @@ declare_deps_provider(
     deps = [
         "@com_google_protobuf//:protobuf_java",
         "@scala_proto_rules_protoc_bridge",
+        "@scala_proto_rules_protoc_gen",
```

Comment:
Is this really needed? As I remember `rules_scala` doesn't use helpers from this jar ie `CodeGenApp`.

Reply:
I'm sure i added this because something was giving an error. you can try removing it and seeing what errors